tests/utils.js

```diff
@@ -157,6 +157,16 @@ describe('Utils', () => {
       chai.expect(utils.hideAbsolutePathsInObject(object)).to.deep.equal(object);
     });
 
+    it('should return a plain object when given an error object', function() {
+      const error = new Error('Hello, World!');
+      const hiddenPaths = utils.hideAbsolutePathsInObject(error);
+
+      chai.expect(hiddenPaths).to.deep.equal({
+        message: 'Hello, World!',
+        stack: error.stack
+      });
+    });
+
```

Comment:
Very good idea, I had a sneaking suspicion about paths in error stack traces, and this confirms it: it can't handle them as-is. I think we'll have to treat error stack traces as an edge-case because there are several paths riddled throughout and they start and end with characters that are valid in paths but aren't actually part of the path, namely the parens.

```
Error: Hello, World!
    at Context.<anonymous> (/home/shou/Prog/Javascript/resin-corvus/tests/utils.js:171:21)
    at callFn (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runnable.js:345:21)
    at Test.Runnable.run (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runnable.js:337:7)
    at Runner.runTest (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:444:10)
    at /home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:550:12
    at next (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:361:14)
    at /home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:371:7
    at next (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:295:14)
    at Immediate.<anonymous> (/home/shou/Prog/Javascript/resin-corvus/node_modules/mocha/lib/runner.js:339:5)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
    at processImmediate [as _immediateCallback] (timers.js:596:5)
```

A naive way I see of solving it is to split on parentheses and treat what's in-between as a potential path, and `basename` it.